### DISCUSSION TITLE
Error while installing the foundry-zksync

### DISCUSSION THREAD

**Discussion by talhazullfiqar** [Cyfrin/foundry-full-course-cu#2596] [Q&A]
Anytime i run the command to install the foundry zk-sync on bash i run into an error

![Screenshot 2024-10-01 222201](https://github.com/user-attachments/assets/64aaf020-6206-4995-838c-3da4edc40abe)

**Comment by EngrPips** [comment]
Hello @talhazullfiqar, You figured it out?
